lib.rs

```diff
@@ -6,6 +6,9 @@
 //! to the heap for larger allocations.  This can be a useful optimization for improving cache
 //! locality and reducing allocator traffic for workloads that fit within the inline buffer.
 
+#![feature(specialization)]
```

Comment:
Good catch. I was playing around with specialization and accidentally rolled it into this. Fixed!